### ISSUE TITLE
User Guide. Add tip about default styles in postgresql

### ISSUE THREAD

**Issue by alexbruy** [qgis/QGIS-Documentation#478]
Default layer styles as saved by QGIS are failing to restore after a backup:

pg_restore: [archiver (db)] Error from TOC entry 3479; 0 2188138 TABLE DATA layer_styles
pg_restore: [archiver (db)] COPY failed for table "layer_styles": ERROR: invalid XML content
DETAIL: Entity: line 1: parser error : StartTag: invalid element name
<!DOCTYPE qgis PUBLIC 'http://mrcc.com/qgis.dtd' 'SYSTEM'>
^
CONTEXT: COPY layer_styles, line 1, column styleqml: "<!DOCTYPE qgis PUBLIC 'http://mrcc.com/qgis.dtd' 'SYSTEM'>
<qgis version="2.2.0-Valmiera" minimumSca..."

According to http://lists.osgeo.org/pipermail/qgis-user/2014-April/026937.html
one need to set the XML option to DOCUMENT and the restore will work.

Original ticket https://hub.qgis.org/issues/10079

**Comment by junek** [comment]
Does the backup also fail with QGIS 2.8. and higher versions of PostGIS/PostgreSQL ?

**Comment by junek** [comment]
insterted a tip into the user manual

**Comment by R0b1nvm** [comment]
Steps to make it work:
1. Make a PLAIN restore within PgAdmin
2. Open the file within a text editor
3. Change the code `SET xmliotuib = content;` with `SET XML OPTION`
4. Use psql to restore the database


> According to http://lists.osgeo.org/pipermail/qgis-user/2014-April/026937.html one need to set the XML option to DOCUMENT and the restore will work.

This link goes to an 404 error not found. The new link: https://lists.osgeo.org/pipermail/qgis-user/2014-April/024619.html found on https://lists.osgeo.org/pipermail/qgis-user/

**Comment by DelazJ** [comment]
@R0b1nvm Sorry but I'm not sure I get what your message is about. The instructions in docs (note at the end of https://docs.qgis.org/3.22/en/docs/user_manual/introduction/general_tools.html#save-in-database and https://docs.qgis.org/3.22/en/docs/user_manual/managing_data_source/supported_data.html#layer-style-backup) are missing something?

**Comment by R0b1nvm** [comment]
@DelazJ sorry. I wanted to add the hyperlink that was referenced. I spend a couple of hours trying to solve this problem myself even with these solutions it took me forever (couple of hours). I got stuck on replacing the set xml options to document and using psql to restore it sindse the restore wouldn't work with PgAdmin. So I included the steps hoping that if another would stumble on this problem this would help reduce the time to fix the problem or hopefully make it a bit more organised.
